Cargo.toml

```diff
@@ -29,6 +30,7 @@ biscuit = "0.6"
 thiserror = "1"
 validator = { version = "0.16", features = ["derive"] }
 mime = "0.3"
+reqwest-maybe-middleware = "0.2.1"
```

Comment:
This line then should be optional, otherwise this dependency will be added to tree without user notice.

Reply:
The whole point of that crate is to be a slim wrapper over `reqwest` without adding other dependencies by default. It kind of defeats the point to put it behind a feature flag, because it exposes a feature flag to add in the new feature here (middleware support).